core/src/kvs/tikv/mod.rs

```diff
@@ -394,7 +394,7 @@ impl super::api::Transaction for Transaction {
 	where
 		K: Into<Key> + Sprintable + Debug,
 	{
-		// TiKV does not support verisoned queries.
+		// TiKV does not support versioned queries.
```

Comment:
🤦

Reply:
@gsserge please resolve this conversation thread to allow pr to merge :pray: